### ISSUE TITLE
Suggestion: Could you please add same details found into <https://authenticator.cc/docs/en/overview> also in Microsoft Edge Add-ons site ?

### ISSUE THREAD

**Issue by SwimmeRM** [Authenticator-Extension/authenticator.cc#40]
More specifically adding these words too might help to spread 'Authenticator: 2FA Client' diffusion even more from '[Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/detail/authenticator-2fa-client/ocglkepbibnalbgmbachknglpdipeoio)':
"Authenticator Extension is an in-browser One-Time Password (OTP) client. Authenticator Extension supports both Time-Based One-Time Password (TOTP, specified in [RFC 6238](https://tools.ietf.org/html/rfc6238)) and HMAC-Based One-Time Password (HOTP, specified in [RFC 4226](https://tools.ietf.org/html/rfc4226)).

In other words, Authenticator Extension can serve as a drop-in replacement for apps such as Google Authenticator."

Regards
